src/components/Search/SearchHeading.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const tablTitle = (props) => {
```

Comment:
The convention is to use exactly the same name for the component here as the name of the file. So it would be either `SearchHeading`, or rename the file to `TablTitle`. Also remember that the convention is to name components with a capital letter to start.

Personally I would pick `SearchHeading`. I think it's more descriptive, and I like explicit names rather than shorted names like "tabl"

Reply:
Ditto. Done